### DISCUSSION TITLE
Score screen seemingly freezing after some map completions

### DISCUSSION THREAD

**Discussion by natstc25** [ppy/osu#31097] [Q&A]
When playing some maps (this has been very rare, not often that it happens), my game will randomly pause for anywhere from ~5-10 seconds before showing a result screen. 

The image attached is all I see for the time that the game is "paused", I can move the cursor, but not much else. Even pressing escape does not bring up the result screen quicker or exit to song select, just has no functionality when in this waiting stage.

This is just a small bug that I found, if it can be called that, though I'm not sure if it's just the game taking a while to report scores and send them to servers (this would sometimes happen on stable and pp values would take about the same amount of time).

There are also no log files or anything to report, since this isn't a "bug" I guess.

Sorry if this is kind of stupid, just don't know if this is an actual bug or the server taking a while to receive the score, such as mentioned before what happens in stable.

![osu_2024-12-12_15-09-21](https://github.com/user-attachments/assets/c42dec79-765e-4167-9943-7961221909e9)

**Comment by peppy** [comment]
This is likely the delay from score submission as you guessed. Eventually we'll make this more asynchronous (backgrounded) so it doesn't pause like this.
